micro-ui/README.md

```
@@ -1,140 +1,273 @@
+# Optional: Feature Flags
+ENABLE_MDMS_BULK_UPLOAD=true
+ENABLE_JSON_EDIT=true
+ENABLE_MDMS_BULK_DOWNLOAD=true
+
+# Optional: Module Configuration
+CORE_UI_MODULE_LOCALE_PREFIX=custom_prefix
+OVERRIDE_ROOT_TENANT_WITH_LOGGEDIN_TENANT=true
+```
 
-`REACT_APP_PROXY_API` ::  `{{server url}}`
+### Sample Configuration Files
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)